src/utils/index.js

```diff
@@ -1,8 +1,15 @@
-import axiosWithAuth from "./axiosWithAuth";
-import useForm from './useForm'
-import useFinder from './useFinder'
-
-export { axiosWithAuth, useForm, useFinder };
-
-
-
+import {
```

Comment:
understand what you did and how, but no clue what it changed/impacted

Reply:
Prettier.